Spanner/src/TransactionalReadTrait.php

```diff
@@ -263,6 +264,8 @@ public function execute($sql, array $options = [])
         $selector = $this->transactionSelector($options, $this->options);
 
         $options['transaction'] = $selector[0];
+        $options += $this->pluckArray(['tags'], $this->options);
```

Comment:
`pluck` and `pluckArray` have a side effect of removing the keys from the input array. So the tags will only be available in `$this->options` the first time you pluck them.

Use an `if (isset($this->options['tags']))` conditional instead to set the tags in the request options.